linuxNew/Jenkinsfile

```diff
@@ -60,7 +60,7 @@ def buildAndTest() {
             break;
     }
     sh("./gradlew packageJdk${DISTRO} checkJdk${DISTRO} --parallel -PPRODUCT=${PRODUCT} -PPRODUCT_VERSION=${VERSION}")
-    archiveArtifacts artifacts: '**/*/build/ospackage/*,**/*/build/reports/**'
+    archiveArtifacts artifacts: '**/*/build/ospackage/*,**/*/build/reports/**,*,**/ospackage/adoptium-ca-certificates*.deb'
```

Comment:
I'm confused about this change. `**/*/build/ospackage/*` has archived everything under `ospackage`. Should no need to archive `ospackage/adoptium-ca-certificates*.deb'`? Also `*` means archived everything, is this intended?

Reply:
Yeah, this was in error, @sxa is working on a new PR to correct it.